lambda-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sailplane/lambda-utils",
-  "version": "3.0.0",
+  "version": "4.0.0-beta.4",
```

Comment:
beta.4 currently available in NPM repository. Once PR is merged, this will be published as 4.0.0.